runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -713,6 +713,26 @@ public void scrollBy(int x, int y) {
         mContentView.scrollBy(x, y);
     }
 
+    public int computeHorizontalScrollRange() {
```

Comment:
in XWALK-4394, I saw it related to XWALK-5417 which description as TC dev for feature XWALK-4394. So the test case is not include.